Assets/Scripts/Layers/LayerTypes/GeoJSONLineLayer.cs

```diff
@@ -47,41 +37,36 @@ public void AddAndVisualizeFeature<T>(Feature feature, CoordinateSystem original
             if (SpawnedVisualisations.Any(f => f.feature.GetHashCode() == feature.GetHashCode()))
                 return;
 
-            var newFeatureVisualisation = new FeatureLineVisualisations() { feature = feature };
+            var newFeatureVisualisation = new FeatureLineVisualisations { feature = feature };
 
-            // Create visual with random color if enabled
-            lineRenderer3D.LineMaterial = GetMaterialInstance();
+            ApplyStyling();
 
             if (feature.Geometry is MultiLineString multiLineString)
             {
-                var newLines = GeoJSONGeometryVisualizerUtility.VisualizeMultiLineString(multiLineString, originalCoordinateSystem, lineRenderer3D);
+                var newLines = GeometryVisualizationFactory.CreateLineVisualisation(multiLineString, originalCoordinateSystem, lineRenderer3D);
                 newFeatureVisualisation.lines.AddRange(newLines);
             }
             else if(feature.Geometry is LineString lineString)
             {
-                var newLine = GeoJSONGeometryVisualizerUtility.VisualizeLineString(lineString, originalCoordinateSystem, lineRenderer3D);
+                var newLine = GeometryVisualizationFactory.CreateLineVisualization(lineString, originalCoordinateSystem, lineRenderer3D);
                 newFeatureVisualisation.lines.Add(newLine);
             }
 
             SpawnedVisualisations.Add(newFeatureVisualisation);
         }
 
+        public void ApplyStyling()
+        {
+            lineRenderer3D.LineMaterial = GetMaterialInstance();
+        }
+        
         private Material GetMaterialInstance()
         {
-            Material featureMaterialInstance;
-            // Create material with random color if randomize per feature is enabled
-            if (RandomizeColorPerFeature)
+            var strokeColor = LayerData.DefaultSymbolizer.GetStrokeColor() ?? Color.white;
+            return new Material(lineRenderer3D.LineMaterial)
             {
-                var randomColor = UnityEngine.Random.ColorHSV();
-                randomColor.a = LayerData.Color.a;
-
-                featureMaterialInstance = new Material(lineRenderer3D.LineMaterial) { color = randomColor };
-                return featureMaterialInstance;
-            }
-
-            // Default to material with layer color
-            featureMaterialInstance = new Material(lineRenderer3D.LineMaterial) { color = LayerData.Color };
-            return featureMaterialInstance;
+                color = strokeColor
```

Comment:
Dit is goed om ergens te documenteren, want als we willen dat het mogelijk is voor lijnen om een outline te hebben, dan zou dit de fill color moeten zijn. maar dat hebben we nu nog niet, dus is strokeColor voor lijnen, en fillColor voor polygons en punten.

Reply:
Outlines zijn vaak andere properties (zoals box-shadow in CSS, en in SymCore was het weer wat anders). Ik zou dus niet de stroke en fill verwisselen. Dat gezegd hebbende, er is behoorlijk wat documentatie om hierover op te leveren.. Ik denk dat een toelichting op de keuze voor kleur properties opgenomen kan wordne in een een styling sectie over (vector) lijnen